src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -62,11 +62,19 @@ public function find($includeArgs = true)
             }
         }
 
-        $dirs = array(PHP_BINDIR);
+        $dirs = array();
         if ('\\' === DIRECTORY_SEPARATOR) {
             $dirs[] = 'C:\xampp\php\\';
         }
 
+        $name = 'php';
+        foreach (array('', '.exe', '.bat', '.cmd', '.com') as $suffix) {
```

Comment:
reading this, it feels like this logic belongs to the "executableFinder", isn't it?

Reply:
Yes, i had changed the executable finder component first to allow looking for executables ONLY in given direcories (currently it is only able to use the given directories as fallback if the executable is not found in the environment path), but you said that it should be reduced to the strict minimum. So the question is: Where should we out this functionality for the bugfix? In long term it would make sense to extend the executable finder with this functionality.